src/DnsSecStack.ts

```diff
@@ -21,6 +29,34 @@ export class DnsSecStack extends cdk.Stack {
       parameterName: Statics.accountDnsSecKmsKey,
     });
 
+    if (props.enableDnsSec) {
+      this.enableDnsSecForAccountRootZone(dnssecKey.keyArn);
+    }
+
+  }
+
+  /**
+   * Enable DNSSEC usign the KMS key from this stack for the account root hosted zone.
+   * @param keyArn
+   */
+  enableDnsSecForAccountRootZone(keyArn: string) {
```

Comment:
Dit draait nu nog niet dus?

Reply:
Vanuit dit project wordt tot voor deze PR een hosted zone aangemaakt (in de sandbox), hiervoor was nog geen DNSSEC nodig. Ik wil eerst deze zone deployen de stap hierna wordt DNSSEC enabelen.